src/main/java/com/powsybl/openloadflow/util/LfHvdcUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.powsybl.openloadflow.util;
+
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.HvdcConverterStation;
+import com.powsybl.openloadflow.network.LfNetwork;
+
+public final class LfHvdcUtils {
+
+    private LfHvdcUtils() {
+    }
+
+    public static boolean isHvdcDandlingInIidm(HvdcConverterStation<?> station, LfNetwork network) {
+
+        if (isIsolated(station.getTerminal().getBusBreakerView().getBus(), network)) {
+            return true;
+        } else {
+            return station.getOtherConverterStation().map(otherConverterStation -> {
+                Bus bus = otherConverterStation.getTerminal().getBusView().getBus();
+                return isIsolated(bus, network);
+            }).orElse(true); // it means there is no HVDC line connected to station
+        }
+    }
+
+    private static boolean isIsolated(Bus bus, LfNetwork network) {
+        if (bus == null) {
+            return true;
+        }
+        if (network != null && network.getBusById(bus.getId()) != null) {
```

Comment:
Can we just explain me why we can be in that case: line 28 to 30?

Reply:
@annetill  this is explained in comment line 29.
The bus is in the LfNetwork. And the connectivity is computed dynamically by the enabled status of the LfBus as contingencies are applied, or actions.

Reply:
I expect the case happens in test  testHvdcDisconnectedThenConnectedByStrategy(String testType)  from the iniital PR https://github.com/powsybl/powsybl-open-loadflow/pull/957 
I will add a similar use case in this PR to be sure it works as expected.